trusted-committer/03-keeping-the-community-healthy.asciidoc

```diff
@@ -1,80 +1,76 @@
+code and code health only. To ensure success in the long run, Trusted Committers should
+strive to keep the community building the software healthy
+as well. Because of that, they must strike a good balance between ensuring product quality and growing a healthy community.
 
 What does a healthy community look like? Quite simply, in a healthy community,
-https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] tend to stick around, can spend most of their time on developing
-software and are able to level up their abilities. As a result, a healthy
-community will also be growing.
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] tend to stick around, can spend most of their time developing software, and are able to level up their abilities.
+As a result, a healthy community will be continually growing.
 
 Why do https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] join and stick around in a community? Some do because they
 subscribe to the purpose or the mission of the community. It is the Trusted Committer's job to
 clearly articulate and promote this purpose. The importance of this is often
-not recognized - marketing a community and their products is truly essential.
+not recognized, but marketing a community and its products is truly essential.
 
-Another, more obvious reason for people to stick around is that they enjoy
-working with other members of the community, including the Trusted Committers. In my
-experience, what this comes down to is that community members treat and
-communicate with each other with utmost respect. Contributions are treated as
-like a gift or a donation, rather than something that detracts from their own
-work and they laud excellent and especially first contributions. The Trusted Committer's job
-in all this is primarily to set an example for others, similarly to setting an
-example for the level of software quality that is expected.  If necessary, the
-Trusted Committers are the ones who should create and enact a code of conduct for the
-community. Should there be community members whose behavior is detrimental or
-even toxic to the community health, it is the Trusted Committer's responsibility to either
-try and change or contain this or, in the worst case, to remove people from the
-community. Trusted Committers should create opportunities for people to get together
-regularly and get to know each other personally.
+Another, more obvious reason for people to stick around is that they
+enjoy working with other members of the community, including the Trusted Committers. A thriving community is one where members treat
+and communicate with each other with the utmost respect. Contributions are
+treated like gifts or donations, and excellent (especially
```

Comment:
Suggested change to line 21: 
Contributions are treated like gifts or donations rather than distractions, and excellent (especially first) contributions are lauded.

Reply:
I've changed this to a commit suggestion making it easier to see the actual difference. 
I think this phrasing is good. I thought about how to explicitly include the priority treatment/inspection of contributions - but figured that the metaphor of the gift hopefully should work with everyone.